### ISSUE TITLE
Iterate IncLim research plan and discussion guide for recruitment launch

### ISSUE THREAD

**Issue by wesrowe** [department-of-veterans-affairs/va.gov-cms#12248]
## Description

Goal for this ticket: get all research materials ready to launch recruiting. (Midpoint review delay could shift that feedback cycle to another ticket.)

## Acceptance Criteria
- [x] Get additions from my notes from meetings on 3/2, 3/1, 2/23
- [x] Get feedback from Content & IA and incorporate
- [x] Update research plan and conversation guide with current timeline 
- [x] Share research plan and conversation guide with Dave Conlon to review
- [x] Get feedback from Dave Conlon and incorporate it
- [x] Get approval from Dave Conlon and add to bottom of research plan and discussion guide
- [x] Get updated prototype link from Jordan into research plan and conversation guide
- [x] Get feedback from Midpoint review and incorporate it (if possible in-sprint)
- [x] Update dates in research plan for research, pilots, and kickoff meeting

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [x] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by cindymerrill** [comment]
Waiting for feedback from Sitewide Content team

**Comment by cindymerrill** [comment]
Reviewed all the feedback from Sitewide Content, responded to comments and added more, which Danielle answered. Discussed all the feedback with Wes, and planning to talk with Jordan when she gets back on Monday. 

Revised my conversation guide a bit based on some of the changes, revised the dates in my research plan by a week to match the current timeline, and then shared both docs with Dave C to review if he has time before we meet with him on Tuesday.

**Comment by cindymerrill** [comment]
BLOCKED by Content

**Comment by jilladams** [comment]
High priority and should be tagged as such as soon as it's unblocked and pulled back into a sprint. (Untagging for now, as it's making other items hard to manage in the Stretch lane.)

**Comment by cindymerrill** [comment]
Revised research plan and conversation guide per recent meetings, and shared with Dave Conlon in case he has time to  review it before our Tuesday meeting with him

**Comment by cindymerrill** [comment]
Great conversation just now, @davidconlon ! Here's the ticket where you can document your approval of my research plans, with the small changes we discussed.

**Comment by davidconlon** [comment]
Great work @cindymerrill! Approved.

**Comment by cindymerrill** [comment]
Completed revisions to research plan and conversation guide, per Dave's feedback.

**Comment by jilladams** [comment]
Pulled into Sprint 79 as a goal. FYI @wesrowe @davidconlon

**Comment by cindymerrill** [comment]
Got prototype link from Jordan, reviewed it and provided feedback, and copied link into research plan and conversation guide.

**Comment by cindymerrill** [comment]
Had Midpoint Review this morning, incorporated a few suggestions into my research plan and convo guide, and then shared them with Shane to launch recruiting
